src/kernel-install/90-loaderentry.install

```diff
@@ -88,7 +90,8 @@ install -g root -o root -m 0644 "$KERNEL_IMAGE" "$ENTRY_DIR_ABS/linux" || {
 }
 
 shift "$INITRD_OPTIONS_SHIFT"
-for initrd; do
+# All files listed as arguments, and staged files called "initrd*" are installed as initrds.
+for initrd in "$@" "${KERNEL_INSTALL_STAGING_AREA}"/initrd*; do
```

Comment:
Isn't the usual name "initramfs-\<version\>.img"? Shouldn't we use that instead of initrd?

Reply:
No, we have a separate directory for the boot loader entry, so the names are usually `linux` and `initrd`. Longer names are used when everything is squished into the same directory, which we are not doing here.